server/leafnode.go

```diff
@@ -158,7 +158,7 @@ func (s *Server) solicitLeafNodeRemotes(remotes []*RemoteLeafOpts) {
 				s.Errorf("LeafNode Remote Credentials file %q malformed", creds)
 			} else if _, err := nkeys.FromSeed(items[1][1]); err != nil {
 				s.Errorf("LeafNode Remote Credentials file %q has malformed seed", creds)
-			} else if uc, err := jwt.DecodeUserClaims(string(items[0][1])); err != nil {
+			} else if uc, err := jwt.DecodeUserClaims(bytesToString(items[0][1])); err != nil {
```

Comment:
The problem I see with that is that - although we own the jwt library - we then must ensure that the string passed to those public APIs would never "retain" (store in some internal structures) the passed string. This is opening the door to issues down the line if one day someone makes a change in such "extern" library without knowing the implications.

Reply:
We own the buffer here as well, so worse case would be a wider buffer hold if they used the string. But not critical here, so will revert.